gensim/models/deprecated/old_saveload.py

```diff
@@ -31,7 +31,7 @@
 
 from six import iteritems
 
-from smart_open import smart_open
+from smart_open import open
```

Comment:
I thought we decided [here](https://github.com/RaRe-Technologies/gensim/pull/2530#discussion_r296324334) to use utils.open consistently. Why are we still using this import?